test/prism/errors/command_calls_31.txt

```diff
@@ -0,0 +1,12 @@
+true && not true
+            ^~~~ expected a `(` after `not`
+            ^~~~ unexpected 'true', expecting end-of-input
+
+true || not true
+            ^~~~ expected a `(` after `not`
+            ^~~~ unexpected 'true', expecting end-of-input
+
+true && not (true)
+            ^ expected a `(` after `not`
+            ^ unexpected '(', expecting end-of-input
+
```

Comment:
I wanted to add a test to confirm that

```
true && not
true
```

be rejected, but I am not sure how to write the test.